INSTALL.md

```diff
@@ -25,7 +25,22 @@ by the Helm charts will expect to be present.
    including a shared read-write volume. See the "Volumes" section below.
 * Ingress controller
    * Because different Kubernetes clusters use different Ingress configurations,
-   the Helm charts provide Nginx ingress templates only. By default, *ingress.create* is set to false, an ingress should be created manually after the chart is installed
+   the Helm charts provide Nginx ingress object templates only. By default, *ingress.create* is set to false, an ingress should be created manually after the chart is installed
```

Comment:
It is not clear what will actually happen when people set `ingress.create` to `true`. We can provide some details for that workflow as it might be used in the testing scenario but I think emphasising that customers should create Ingress manually is a good approach.

Reply:
> @dylanrathbone I love the new diagram now :) As to endpoints, yeah, perhaps it's too many details, and it's not really necessary.

Yup, I want this to be more about the Ingress and not about the other interacting concepts too. I actually think this current diagram is a bit overboard too, but I'll leave as is for now.

Reply:
> It is not clear what will actually happen when people set `ingress.create` to `true`. We can provide some details for that workflow as it might be used in the testing scenario but I think emphasising that customers should create Ingress manually is a good approach.

I agree, let me update the wording around this so that its more clear.